pubspec.yaml

```diff
@@ -1,10 +1,10 @@
 name: http2
 version: 1.0.2-dev
 description: A HTTP/2 implementation in Dart.
-homepage: https://github.com/dart-lang/http2
+repository: https://github.com/dart-lang/http2
```

Comment:
Are we going to be changing this everywhere? Why?

Reply:
It more clearly represents the content - https://dart.dev/tools/pub/pubspec#homepage

Not worth doing one-off fixes everywhere, but something I do if I'm touching a pubspec